lib/operations/common/BitBucketServerRepoRef.ts

```diff
@@ -113,11 +116,25 @@ export class BitBucketServerRepoRef extends AbstractRemoteRepoRef {
                                   title: string,
                                   body: string,
                                   head: string,
-                                  base: string): Promise<ActionResult<this>> {
+                                  base: string,
+                                  reviewers?: Array<{type: Exclude<PullRequestReviewerType, PullRequestReviewerType.team>, name: string}>,
+    ): Promise<ActionResult<this>> {
         const url = `${this.scheme}${this.apiBase}/${this.apiPathComponent}/pull-requests`;
         logger.debug(`Making request to '${url}' to raise PR`);
         const repoId = await this.getRepoId(creds);
-        const reviewers = await this.getDefaultReviewers(creds, repoId, head, base);
+
+        // Figure out reviewers
+        const allReviewers = await this.getDefaultReviewers(creds, repoId, head, base);
+        if (reviewers) {
+            if (reviewers.some(r => r.type !== PullRequestReviewerType.individual)) {
+                throw new Error(`Bitbucket only supports reviewer type of individual!  Found ` +
+                    JSON.stringify([...new Set(reviewers.map(r => PullRequestReviewerType[r.type]))]));
+            }
+            const ids = reviewers.filter(r => r.type === PullRequestReviewerType.individual).map(fr => fr.name);
```

Comment:
Is the filter here necessary since you throw the error above? Perhaps the error should just be a warning?

Reply:
Will update; as you mention it can be removed.